codespell_lib/data/dictionary.txt

```diff
@@ -45303,6 +45303,7 @@ re-uplods->re-uploads
 re-usable->reusable
 re-use->reuse
 re-useable->reusable
```

Comment:
While w're at it, I suggest adding:
```
re-used->reused
re-uses->reuses
```

Reply:
I've done so. Thanks for the suggestion!